jwcrypto/jws.py

```diff
@@ -8,6 +8,17 @@
 from jwcrypto.jwa import JWA
 from jwcrypto.jwk import JWK
 
+__all__ = [
+    'JWSHeaderParameter',
+    'JWSHeaderRegistry',
+    'jws_default_allowed_algs',
+    'InvalidJWSSignature',
```

Comment:
same here also JWSCore is not really public

Reply:
I agree